POSTINSTALL-README.md

```diff
@@ -48,7 +48,9 @@ Documentation is available [here](https://github.com/secureblue/secureblue/blob/
 
 Setting a GRUB password helps protect the device from physical tampering and mitigates various attack vectors, such as booting from malicious media devices and changing boot or kernel parameters.
 
-To set a GRUB password, use the following command. By default, the password will be required when modifying boot entries, but not when booting existing entries.
```

Comment:
This seems fine how it was

Reply:
if you think so. If I "resolve" this, will it be changed automatically?

Reply:
no